application-workloads/wordpress/wordpress-app-service-mysql-inapp/main.bicep

```diff
@@ -0,0 +1,77 @@
+@description('SKU (App Service Pricing Tier)')
+@allowed([
+  'F1'
+  'B1'
+  'S1'
+])
+param sku string = 'F1'
+
+@description('GitHub repo to deploy to App Service')
+param repoUrl string = 'https://github.com/azureappserviceoss/wordpress-azure'
```

Comment:
you cannot link to a personal repo in a sample - you can use a public repo/sample though